include/sqlite_orm/sqlite_orm.h

```diff
@@ -6161,22 +6161,6 @@ namespace sqlite_orm {
 
 // #include "../serialize_result_type.h"
 
-#ifdef SQLITE_ORM_STRING_VIEW_SUPPORTED
```

Comment:
why this is disappeared?

Reply:
Because the incuded file serialize_result_type.h appears already on line [4120](https://github.com/fnc12/sqlite_orm/blob/1895b18832b4b36da82b14c3ddbcf532ba01f53a/include/sqlite_orm/sqlite_orm.h#L4120).

You just didn't notice until this PR :)

Reply:
oh nice. Thanks